src/routes/docs/tutorials/astro-ssr-auth/step-3/+page.markdoc

```diff
@@ -29,7 +29,6 @@ export const SESSION_COOKIE = "my-custom-session";
 // Admin client, used to create new accounts
 export function createAdminClient() {
   const client = new Client()
-    .setEndpoint(import.meta.env.PUBLIC_APPWRITE_ENDPOINT)
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?